### ISSUE TITLE
Couldn't raise all-in

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#739]
Fix so we can raise all-in.

```
Dealing #226138: 400 500 Pot-Limit Mexican (High only)
Everyone antes 400
1 pokerchimp 1 9100 400 [ ] 3d
2 gerdog 2 17650 400 [ ] 3s
3 B > JADC🚴 0 12000 400 [ ] 6h
4 🍺 deadhead 2 14650 400 [ ] 2h
My cards are: [BG] 2h
JADC🚴 brings-in for 500
It is 500 to you
🍺 deadhead raises 1300 to 1800
pokerchimp folds
gerdog calls
JADC🚴: a 6?
JADC🚴 raises 6500 to 8300
It is 6500 to you
You did not raise. You wanted to raise 4550 but the minimum is 6500
You did not raise. You wanted to raise 4550 but the minimum is 6500
🍺 deadhead asked for and gets 40 more seconds
🍺 deadhead calls
```

**Comment by ctm** [comment]
When using `force-status` I got into a situation where `pokerchimp` had a `Raise To` button whose value was too low to put her all-in, but if I chose `max`, it created the right value and the server allowed it.

Reproducing this with `force-status` is a PITA, because there's no guarantee `JADC` will get the bring-in. I didn't think that would matter much, but I'm not convinced the error I mention in my preceding paragraph is the same error that bit me yesterday evening.
